tests/background/tab-state-test.js

```diff
@@ -134,15 +134,19 @@ describe('TabState', () => {
   describe('#updateAnnotationCount', () => {
     let clock;
     let getStub;
+    let uriForBadgeRequestStub;
     const INITIAL_WAIT_MS = 1000;
     const MAX_WAIT_MS = 3000;
+    const CACHE_EXPIRATION_MS = 3000;
 
     beforeEach(() => {
       clock = sinon.useFakeTimers();
       getStub = sinon.stub();
+      uriForBadgeRequestStub = sinon.stub();
```

Comment:
```suggestion
      uriForBadgeRequestStub = sinon.stub().returnsArg(0);
```

Rather than configuring this stub's behavior in many separate tests, it looks like it would be simpler to do it here.

Reply:
Yes, I agree. I tried but then I couldn't overwrite it later.

Reply:
I have tried again and it is working. Thanks for the suggestion.